### PR TITLE
re #251 - Removed (balanace) from Nashville-Davidson

### PR DESCRIPTION
**Pull request by stepps00**

### CODE REVIEW

**Comment by nvkelso** [comment]
I'm assuming this is also true:

```
Don't modify original qs:loc and qs:loc_alt properties.
```

**Comment by nvkelso** [comment]
Recommend:
`wof:name`: **Nashville**

But add:
 `wof:label`: **Nashville-Davidson County**

File: data/101/723/183/101723183.geojson [hunk 1]
```diff
@@ -51,7 +51,8 @@
     ],
     "wof:id":101723183,
     "wof:lastmodified":1452293124,
-    "wof:name":"Nashville-Davidson",
+    "wof:label":"Nashville-Davidson County"
```
Comment (nvkelso) [inline data/101/723/183/101723183.geojson]: Needs a trailing comma.

Comment (stepps00) [inline data/101/723/183/101723183.geojson]: Added in the trailing comma after `"Nashville-Davidson County"`

**Comment by nvkelso** [comment]
LGTM, ready for merging.

**Pull request merged**